.travis.yml

```diff
@@ -62,6 +62,30 @@ jobs:
         - CXX=g++-9
         - CC=gcc-9
         - BUILD_TYPE=Release
+    - os: linux
+      name: "Documentation"
+      addons:
+        apt:
+          sources:
+            - sourceline: 'ppa:ubuntu-toolchain-r/test'
+          packages:
+            - g++-9
+            - doxygen
+            - doxygen-doc
+            - doxygen-latex
+            - graphviz
+      script:
+        - doxygen --version
+        - doxygen -u doc/Doxyfile
+        - make -j 2 doc 2>doxygen_warnings.txt
```

Comment:
Do you think we need multiple cores? The documentation is really small. Or will it get bigger?

Reply:
Yes, the documentation is very small because it is just a template and in fact we currently do not save run time here. But I hope that developers will use this code for larger projects. For instance, in the SeqAn builds it makes a difference to build documentation multi-threaded.